test_communication/package.xml

```diff
@@ -11,21 +11,24 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>ament_cmake_auto</build_depend>
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>rosidl_default_generators</build_depend>
+
   <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Because `ament_cmake_auto` depends on `ament_cmake` and exports that dependency (https://github.com/ament/ament_cmake/blob/7b31d859f3d42b9eb6c84db807d95a5604a6881b/ament_cmake_auto/ament_cmake_auto-extras.cmake#L17).